iac-extensions/docker/src/test/resources/checks/ClearTextProtocolDowngradeCheck/wget.dockerfile

```diff
@@ -32,5 +32,22 @@ RUN wget ${OPENSSL3_URL}
 # Noncompliant@+1
 RUN wget ${OPENSSL3_URL_2}
 
+# Noncompliant@+1
+RUN wget --secure-protocol=TLSv1_2 -q -O - https://might-redirect.example.com/install.sh
+# Noncompliant@+1
+RUN wget --secure-protocol=TLSv1_2 https://might-redirect.example.com/install.sh -q -O -
+# Noncompliant@+1
+RUN wget --secure-protocol=TLSv1_2 -q -O - -- https://might-redirect.example.com/install.sh
+
+RUN wget --max-redirect=0 --secure-protocol=TLSv1_2 -q -O - https://might-redirect.example.com/install.sh
+RUN wget --secure-protocol=TLSv1_2 -q -O -  --max-redirect=0 https://might-redirect.example.com/install.sh
+RUN wget --secure-protocol=TLSv1_2 https://might-redirect.example.com/install.sh -q -O - --max-redirect=0
+RUN wget --secure-protocol=TLSv1_2 https://might-redirect.example.com/install.sh --max-redirect=0 -q -O -
+RUN wget --secure-protocol=TLSv1_2 -q -O - --max-redirect=0 -- https://might-redirect.example.com/install.sh
```

Comment:
Please add examples for `--max-redirect='0'` and `--max-redirect="0"` - probably a small adjustment in detection logic will be needed.

Reply:
Not sure what would be the proper way to support this (maybe some logic in the CommandDetector? Support for flags with values ignoring quotes on values?), so I added them as additional flag variants.